### ISSUE TITLE
Unable to close rewarded video ad with screen reader

### ISSUE THREAD

**Issue by impure** [googleads/googleads-mobile-flutter#196]
Using talkback on an Android phone it is not possible to select the close button on a rewarded video ad.

Plugin version: 0.12.1+1

<details>
  <summary>Logs</summary>

```
[√] Flutter (Channel stable, 2.0.5, on Microsoft Windows [Version 10.0.19041.928], locale en-US)
    • Flutter version 2.0.5 at C:\Flutter
    • Framework revision adc687823a (9 days ago), 2021-04-16 09:40:20 -0700
    • Engine revision b09f014e96
    • Dart version 2.12.3

[√] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at C:\Users\impur\AppData\Local\Android\sdk
    • Platform android-30, build-tools 29.0.3
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files (x86)\Google\Chrome\Application\chrome.exe

[√] Android Studio (version 4.1.0)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[√] IntelliJ IDEA Ultimate Edition (version 2020.2)
    • IntelliJ at C:\Program Files\JetBrains\IntelliJ IDEA 2019.3.3
    • Flutter plugin version 50.0
    • Dart plugin version 202.7319.5

[√] VS Code (version 1.49.1)
    • VS Code at C:\Users\impur\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.15.1

[√] Connected device (2 available)
    • Chrome (web) • chrome • web-javascript • Google Chrome 90.0.4430.85
    • Edge (web)   • edge   • web-javascript • Microsoft Edge 90.0.818.46

• No issues found!
```

</details>

**Comment by maheshmnj** [comment]
I am able to reproduce the issue using the latest `google_mobile_ads:^ 0.13.3` with talback on the rewarded ad cannot be closed looks like the button is disabled when the timer runs out.This issue only happens when talkback is on.

<details>
<summary>output</summary>

https://user-images.githubusercontent.com/31410839/129044038-e39298ce-54f4-403d-b28b-05da7370db15.mov

</details>

<details>
<summary>flutter doctor -v</summary>

```
[✓] Flutter (Channel stable, 2.2.3, on macOS 11.4 20F71 darwin-arm, locale en-GB)
    • Flutter version 2.2.3 at /Users/mahesh/Documents/flutter
    • Framework revision f4abaa0735 (4 weeks ago), 2021-07-01 12:46:11 -0700
    • Engine revision 241c87ad80
    • Dart version 2.13.4

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/mahesh/Library/Android/sdk
    • Platform android-30, build-tools 30.0.3
    • ANDROID_HOME = /Users/mahesh/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)

[✓] IntelliJ IDEA Community Edition (version 2021.1.2)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 57.0.5
    • Dart plugin version 211.7233

[✓] VS Code (version 1.58.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.24.0

[✓] Connected device (4 available)
    • Redmi K20 Pro (mobile)                       • android-arm64  • Android 11 (API 30)
    • iPhone 12 Pro Max (mobile) • 6A434744-B348-4FB3-AB10-131292174456 • ios            • com.apple.CoreSimulator.SimRuntime.iOS-14-4 (simulator)
    • macOS (desktop)            • macos                                • darwin-arm64   • macOS 11.4 20F71 darwin-arm
    • Chrome (web)               • chrome                               • web-javascript • Google Chrome 92.0.4515.107

• No issues found!
```

</details>

**Comment by Francesco-FL** [comment]
Same thing for interstitials, I can't get out of the ad with talkback.

If I understood the target system correctly and p3-low means essentially not very important, I would like to ask you to prioritize this bug.
I'm not visually impaired, but blocking the user like this means they have to uninstall the app because it becomes unusable.

In my use case for example, when a user clicks a main button, I first show the interstitial and then execute the operation, but if you can't exit the interstitial, the application becomes unusable and this means uninstall and potential (honest) one star review.

plugin version: 2.2.0
test device: Motorola Moto g22

https://user-images.githubusercontent.com/83022092/215098978-91febe69-ef8b-4561-bf67-c0a30e8f7b6b.mp4

**Comment by huycozy** [comment]
We’re closing this issue due to inactivity. If you’re still impacted, please create a new issue via the [Developer Forum](https://groups.google.com/forum/#!categories/google-admob-ads-sdk).
